config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :mechanics, only: [:show]
+  resources :amusement_parks, only: [:show]
+  resources :ride_mechanics, only: [:create]
```

Comment:
While this last route is technically "restful" because you are using the `resources` syntax, the way you are using it is incorrect. The way you are using it is going to provide a route like this: `POST "/ride_mechanics", to: "ride_mechanics#create"` instead of what the route should be, which is this: `POST "/mechanics/:id/rides", to: "ride_mechanics#create"`.

The way you have it, in order to associate a mechanic and a ride, you are going to have to jerry-rig the mechanic ID in somehow, either through query params, a hidden field, or some other way. You would not have to do this if the route path was correct, since the mechanic ID would be in the path.

```ruby
resources :mechanics, only: [:show] do
  resources :rides, only: [:create], controller: "ride_mechanics"
end

resources :amusement_parks, only: [:show]
```